book/src/kubernetes-changelog.md

```diff
@@ -3,6 +3,33 @@
 This page summarizes major CSI changes made in each Kubernetes release. For
 details on individual features, visit the [Features section](features.md).
 
+
+## Kubernetes 1.28
+
+### Features
+* GA
+    * [Automatic retroactive assignment of a default StorageClass](https://kubernetes.io/blog/2023/08/18/retroactive-default-storage-class-ga/)
+    * [Non-graceful node shutdown](https://kubernetes.io/blog/2023/08/16/kubernetes-1-28-non-graceful-node-shutdown-ga/)
+* Alpha
+    * [PersistentVolume last phase transition time](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md)
```

Comment:
Sorry for going back and forth on this.  We discussed this at CSI implementation meeting. Since some of these features do not require any CSI driver changes, it is confusing to add them in this doc.  Can you please remove the following?
- Automatic retroactive assignment of a default StorageClass
- Non-graceful node shutdown
- PersistentVolume last phase transition time

Reply:
Did you submit your commit?  You marked this as resolved but these features are still here in the PR.

Reply:
I screwed up my rebase...

Reply:
Sorry about that mess. Do you want me to create a new PR ?

Reply:
Sure.